tests/runtime/test_http_server.py

```diff
@@ -260,6 +257,25 @@ def test_inference_other_task(other_task_model_id):
         assert json_response["farewell"] == "goodbye: world 42 times"
 
 
+def test_inference_streaming_sample_module(sample_task_model_id):
+    """Simple check that we can ping a model"""
+    server = http_server.RuntimeHTTPServer()
+    with TestClient(server.app) as client:
+        json_input = {"inputs": {"name": "world"}}
+        stream = client.post(
+            f"/api/v1/{sample_task_model_id}/task/server-streaming-sample",
+            json=json_input,
+        )
+        assert stream.status_code == 200
+        assert (
+            stream.content.decode(stream.default_encoding).count("SampleOutputType")
+            == 10
+        )
+        assert (
+            b"SampleOutputType(greeting='Hello world stream')\r\n\r\n" in stream.content
```

Comment:
so many carriage returns!

Reply:
yeah, even the [example](https://github.com/sysid/sse-starlette/blob/master/tests/test_event_source_response.py) from `test_event_source_response` has them